molecule/cookiecutter/scenario/driver/lxd/{{cookiecutter.molecule_directory}}/{{cookiecutter.scenario_name}}/prepare.yml

```diff
@@ -1,11 +0,0 @@
----
```

Comment:
Why was this removed?

Reply:
It is replaced by the expanded `molecule/provisioner/ansible/playbooks/lxd/prepare.yml` as seen a bit further below.

Reply:
Oh, I see I had missed that.  The prepare you created is similar to our Dockerfile setup where we ensure the docker container has the necessary dependencies. 👍